sql/schemas/announcement.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS academic_announcement (
+    date_of_creation  DATE    NOT NULL,
+    title             VARCHAR NOT NULL,
+    title_link        VARCHAR NOT NULL,
```

Comment:
primary key? duplicates?

Reply:
This is an aspect I'd like to have more discussion on: what should be the right primary key for this table?

Maybe (date, title) or (date, title_link) since those are basically what identify each tuple

Reply:
Went with date and title as composite key